.hintrc

```diff
@@ -0,0 +1,19 @@
+{
+    "connector": {
+      "name": "local",
+      "options": {
+        "pattern": ["**", "!.git/**", "!node_modules/**"]
+      }
+    },
+    "extends": ["development"],
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
+      "meta-charset-utf-8",
+      "meta-viewport",
+      "no-inline-styles:error"
+    ]
+  }
```

Comment:
[ OPTIONAL ] Please align the button `Get in touch to the left of the contact form as it's defined in the design for mobile view.
![btn](https://user-images.githubusercontent.com/35504149/129959752-ecba5a8d-cb6e-4a26-916f-acbc7a8d7a2c.png)